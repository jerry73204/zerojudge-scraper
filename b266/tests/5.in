1 8 1
6 6 9 2 1 3 8 6
1
