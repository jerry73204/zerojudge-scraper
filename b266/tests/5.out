1 8
6 6 9 2 1 3 8 6
